src/game.cpp

```diff
@@ -10838,24 +10838,40 @@ void game::wield( int pos )
         add_msg( m_info, _( "You cannot unwield your %s." ), u.weapon.tname().c_str() );
         return;
     }
-    if( pos == INT_MIN ) {
-        pos = inv_for_all( _( "Wield item" ), _( "You have nothing to wield." ) );
+
+    item_location loc;
+    if( pos != INT_MIN ) {
+        loc = std::move( item_location( u, &u.i_at( pos ) ) );
```

Comment:
Pretty sure it was non-copyable from the very start, when I implemented it.
The pseudo-persistence is needed for saves because player actions need that to save their locations.